src/bin/doodle/format/base.rs

```diff
@@ -73,6 +77,9 @@ pub fn is_bytes(bytes: &[u8]) -> Format {
     tuple(bytes.iter().copied().map(is_byte))
 }
 
+/// ByteSet consisting of 0..=127, or the valid ASCII range (including control characters)
+pub const VALID_ASCII: ByteSet = ByteSet::from_bits([u64::MAX, u64::MAX, 0, 0]);
```

Comment:
Temping to say "7-bit ASCII" somehow rather than "valid", specifically... :thinking:

Reply:
Doesn't ASCII imply 7-bit ASCII?

man ascii(7)
> ASCII is the American Standard Code for Information Interchange.  It is a 7-bit code.